src/sqlancer/stonedb/gen/StoneDBExpressionGenerator.java

```diff
@@ -253,7 +256,7 @@ public String getTextRepresentation() {
      */
     public enum StoneDBBinaryLogicalOperator implements Operator {
 
-        AND("AND"), OR("OR"), XOR("XOR");
+        AND("AND"), OR("OR");
```

Comment:
Same, why disable many features?

Reply:
not sure the reason, will not delete now. May be changed if needed in the future.